src/flexbox/nasa.html

```diff
@@ -0,0 +1,51 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="nasa.css">
```

Comment:
get in a habit of using `./nasa.css` instead of `nass.css`. while it works, `./` is the preferred way to reference a link